edx_rest_api_client/client.py

```diff
@@ -81,11 +107,53 @@ def get_oauth_access_token(url, client_id, client_secret, token_type='jwt', gran
     except KeyError:
         raise requests.RequestException(response=response)
 
-    expires_at = now + timedelta(seconds=expires_in)
+    expires_at = now + datetime.timedelta(seconds=expires_in)
 
     return access_token, expires_at
 
 
+# Consider tokens that expire in 5 seconds as already expired
+ACCESS_TOKEN_EXPIRED_THRESHOLD = 5
```

Comment:
Should this be put into settings? Or at least at the top of the file?

Reply:
@dianakhuang: I moved it to the top of the file and added more documentation.  I can add it as a setting, but will follow-up with that to keep moving.